### PR TITLE
[ci skip] adding user @edwinnglabs

### LINKED ISSUE #7: @conda-forge-admin, please add user @edwinnglabs
### Additional comment:

_No response_

### PR DESCRIPTION
**Pull request by conda-forge-daemon**
Hi! This is the friendly automated conda-forge-webservice.

I've added user @edwinnglabs as instructed in #7.

Merge this PR to add the user. Please do not rerender this PR or change it in any way. It has `[ci skip]` in the commit message to avoid pushing a new build and so the build configuration in the feedstock should not be changed.

Please contact [conda-forge/core](https://conda-forge.org/docs/maintainer/maintainer_faq.html#mfaq-contact-core) to have this PR merged, if the maintainer is unresponsive.

Fixes #7

### CODE REVIEW

**Comment by edwinnglabs** [comment]
@sugatoray

**Comment by sugatoray** [comment]
@edwinnglabs This method adds you to the repository as a maintainer but does not show you as one on the README (since CI run is skipped). On the other hand if you add your GitHub username as a maintainer to the recipe, that takes care of everything.

**Comment by edwinnglabs** [comment]
> @edwinnglabs This method adds you to the repository as a maintainer but does not show you as one on the README (since CI run is skipped). On the other hand if you add your GitHub username as a maintainer to the recipe, that takes care of everything.

Yes. I will do that but can you merge this request first?

**Review by edwinnglabs — APPROVED** [review]

**Comment by edwinnglabs** [comment]
@sugatoray it seems you still need to merge this PR. Although the recipe changed, there is a `.github/CODEOWNER` file change missing :(

**Comment by sugatoray** [comment]
Let me take a look at it later.

**Review by edwinnglabs — APPROVED** [review]

**Comment by edwinnglabs** [comment]
> Let me take a look at it later.

Please. I'm rushing for a new version(v1.1.4) for orbit and wants to benchmark this one (v1.1.3) soon.

**Comment by sugatoray** [comment]
Ok. Could you please fork the repo and send another PR. There was some error after the PR was merged last time. A simple new PR with the build number raised by 1 should fix this. 

Please also follow the instructions in the standard PR body.

**Pull request merged**
